### ISSUE TITLE
Quality limit settings are ignored 

### ISSUE THREAD

**Issue by lacalamine64** [jellyfin/jellyfin#8055]
### Please describe your bug

I set up quality limits for external devices in global settings as well for each user but even with 1 mbps set all 4K files try to play in their full glory which won't work with my bandwidth.

In the log below you see that JF sets IsInLocalNetwork: true but that is wrong. I played it over cellular, disconnected from my home network.

It worked before 10.8.x

Tried clients are Web (Safari on iOS), JF Mobile (iOS), Swiftfin (iOS/tvOS).

### Jellyfin Version

Other

### if other:

10.8.1

### Environment

```markdown
- OS: Unraid
- Virtualization: Docker (Official Jellyfin)
- Clients: Browser, iOS: JF Mobile & Swiftfin
- Browser: Safari iOS 15
- FFmpeg Version: 5.x (Official Jellyfin Docker)
- Playback Method: Transcode
- Hardware Acceleration:
- Plugins: -
- Reverse Proxy: nginx
- Base URL: no
- Networking: host
- Storage: local
```

### Jellyfin logs

```shell
[2022-07-04 08:48:21.225 +02:00] [INF] [19] Jellyfin.Api.Helpers.MediaInfoHelper: RemoteClientBitrateLimit: 1000000, RemoteIp: "172.17.0.1", IsInLocalNetwork: True
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
AFAIK the local network recogition (especially usage of X-Forwarded-For headers) was fixed in 10.8, so it might have worked before but does not now. You can check what IP Jellyfin associates with a user via the activity dashboard in the UI.

Make sure to correctly configure `LocalNetworkSubnets` and `KnownProxies` within `/etc/jellyfin/network.xml` or the equivalent location if you use docker. This settings should also be configurable via the UI.

**Comment by lacalamine64** [comment]
Thanks for your tip. JF recognized the external IPs but handled them like internal ones.
After defining IP/Subnet and IP of the proxy via UI it seems to have solved it for me.
